README.md

```diff
@@ -23,6 +23,14 @@
   - [Update a Package](./docs/versioning.md#update-a-package)
   - [Updating a Launchpad Dependency](./docs/versioning.md#updating-a-launchpad-dependency)
 
+### Testing
+
+From the root folder:
+
+```js
+npm run test
```

Comment:
`yarn test`

we are recommending yarn because we keep yarn.lock files for consistency